### ISSUE TITLE
idea: use consistent colours

### ISSUE THREAD

**Issue by ForbesLindesay** [jlongster/live-css-layout#2]
It would be great if it used consistent colours, and maybe also aimed to pick colours that are as distinct as possible?

**Comment by jlongster** [comment]
Definitely, this was just a quick hack. I kind of like the random colors refreshing on each paint, but it would be more helpful to assign colors once and try to make them distinctive. We could even let the user assing a custom color to the node.

Not sure I'm going to develop this much further (at least for now), but I'm absolutely open to pull requests.
